celery.sh issue

Unsure of what to try next...

shaunjohann@DESKTOP:/mnt/c/repo/ix$ make worker
docker-compose run --rm web celery.sh
Creating ix_web_run ... done
exec /usr/bin/ix/celery.sh: no such file or directory
ERROR: 1
make: *** [Makefile:136: worker] Error 1
shaunjohann@DESKTOP-6QNCSD0:/mnt/c/repo/ix$ 


shaunjohann@DESKTOP:/mnt/c/repo/ix$ make bash
docker-compose run --rm web /bin/bash
Creating ix_web_run ... done
root@56bfb0a309b0:/var/app# cd ..
root@56bfb0a309b0:/var# cd ..
root@56bfb0a309b0:/# cd usr/bin/ix
root@56bfb0a309b0:/usr/bin/ix# ls
celery.sh  install_node.sh  ix_asgi.sh  nvm_init.sh  webpack.sh
root@56bfb0a309b0:/usr/bin/ix#

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -48,7 +48,7 @@ services:
   # Agent worker: celery, django
   worker:
     image: ghcr.io/kreneskyp/ix/sandbox:latest
-    command: celery.sh
+    command: celery -A ix worker --loglevel=info --pool=solo --concurrency=1
     links:
       - db
       - redis
@@ -64,7 +64,8 @@ services:
       #- .vault/.client.env
     environment:
       VAULT_BASE_PATH: "ix"
-      DJANGO_SETTINGS_MODULE: "ix.server.settings"
+      DJANGO_SETTINGS_MODULE: "ix.server.celery_settings"
+
 
   # nodejs / frontend builder
   nodejs:

